cli/aws_orbit/cleanup.py

```diff
@@ -208,3 +208,43 @@ def delete_kubeflow_roles(env_stack_name: str, region: str, account_id: str) ->
                 _logger.error("Error. There were concurrent operations in modifying this role")
             except iam_client.exceptions.ServiceFailureException as err:
                 _logger.error(f"Service error: {err}")
+
+
+# Removes ELB since the listener is attached to the target group
+# and the target group can't be removed with a listener attached
+def delete_target_group(env_stack_name: str) -> None:
+    elb_client = boto3_client("elbv2")
+
+    target_groups = elb_client.describe_target_groups()
+
+    for target_group in target_groups.get("TargetGroups"):
+        target_group_arn = target_group.get("TargetGroupArn")
+
+        target_group_tags = elb_client.describe_tags(ResourceArns=[target_group_arn])
+
+        for tag in target_group_tags.get("TagDescriptions")[0].get("Tags"):
```

Comment:
to prevent crashes in case TagDescriptions or Tags don't exist you should make use of the default value of .get(). if a default isn't specified it returns None which would cause an error for the [0] and the for loop

`for tag in target_group_tags.get("TagDescriptions", [{}])[0].get("Tags", [])`